warehouse/common/src/test/java/datawave/common/io/FilesFinderTest.java

```diff
@@ -0,0 +1,115 @@
+package datawave.common.io;
+
+import com.google.common.io.Files;
+
+import datawave.common.test.utils.FileUtils;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.StringJoiner;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import java.util.stream.Stream;
+
+import static datawave.common.test.utils.FileUtils.createTemporaryFile;
+
+public class FilesFinderTest {
+    private static final String TEMP_DIR = Files.createTempDir().getAbsolutePath();
+    private static final String CLASSPATH_DELIMITER = ":";
+    private static final String FILE_PREFIX = "File";
+    
+    private static String TEST_LEVEL0_PROP_FILE;
+    private static String TEST_LEVEL1_JAR_FILE;
+    private static String TEST_LEVEL1_XML_FILE;
+    private static String TEST_LEVEL2_XML_FILE;
+    
+    @BeforeClass
+    public static void setup() throws IOException {
+        TEST_LEVEL0_PROP_FILE = createTemporaryFile(TEMP_DIR, 0, FILE_PREFIX, "properties");
+        TEST_LEVEL1_JAR_FILE = createTemporaryFile(TEMP_DIR, 1, FILE_PREFIX, "jar");
+        TEST_LEVEL1_XML_FILE = createTemporaryFile(TEMP_DIR, 1, FILE_PREFIX, "xml");
+        TEST_LEVEL2_XML_FILE = createTemporaryFile(TEMP_DIR, 2, FILE_PREFIX, "xml");
+    }
+    
+    @AfterClass
+    public static void tearDown() throws IOException {
+        FileUtils.deletePath(TEMP_DIR);
+    }
+    
+    @Test
+    public void testGetFilesFromClasspath() {
+        Collection<String> expectedFiles = Stream.of(TEST_LEVEL1_XML_FILE, TEST_LEVEL2_XML_FILE).sorted().collect(Collectors.toList());
```

Comment:
We could wrap this initializer into a method like:

`Collection<String> expectedFiles = sortedList(TEST_LEVEL1_XML_FILE, TEST_LEVEL2_XML_FILE);`

Reply:
Makes sense.  I will update.   I will see how often I use this in my test cases.